batchflow/tests/model_save_load_test.py

```diff
@@ -0,0 +1,177 @@
+""" Test for model saving and loading """
+# pylint: disable=import-error, no-name-in-module
+
+import pytest
+
+import numpy as np
+
+from batchflow import Pipeline
+from batchflow import B, V, C, I
+
+from batchflow.models.tf import VGG7 as TF_VGG7
+from batchflow.models.torch import VGG7 as TORCH_VGG7
+
+PATH = 'my_mdl'
+BATCH_SIZE = 20
+
+
+@pytest.mark.slow
+@pytest.mark.parametrize(
+    'mode',
+    [pytest.param('static', marks=pytest.mark.skip(reason="doesn't work")),
```

Comment:
Is it not working at all?

Reply:
saving static models is broken, i'm investigating it right now
I will split mode parametrization for load and save pipelines

Reply:
removed `'static'`  after discussion with @roman-kh